sdk/test/AzureTest/Scrubbing.swift

```diff
@@ -36,19 +36,7 @@ public class SubscriptionIDFilter : Filter {
            "operation-location": .remove,
            "azure-asyncoperation": .remove,
            "www-authenticate": .remove,
-           "access_token": .remove,
-            "authorization": .remove,
-            "client-request-id" : .remove,
-            "retry-after" : .remove,
-            "x-ms-client-request-id": .remove,
-            "x-ms-correlation-request-id": .remove,
-            "x-ms-ratelimit-remaining-subscription-reads": .remove,
-            "x-ms-request-id":.remove,
-            "x-ms-routing-request-id": .remove,
-            "x-ms-gateway-service-instanceid": .remove,
-            "x-ms-ratelimit-remaining-tenant-reads": .remove,
-            "x-ms-served-by": .remove,
-            "x-ms-authorization-auxiliary": .remove
+           "access_token": .remove
```

Comment:
Any reason these aren't also in the same place as the other header removals?

Reply:
It's because those headers are listed with the original Subscription Id issue so I handled them along with the IDs.

Reply:
If you think they'd be better placed with the others I can move them over quick.

Reply:
Yes, even in that issue, they are separate bullets from Subscription ID.